VTFLib/VTFFile.cpp

```diff
@@ -3278,13 +3307,16 @@ vlBool CVTFFile::CompressDXTn(vlByte *lpSource, vlByte *lpDest, vlUInt uiWidth,
 	srcTexture.dwDataSize = uiHeight * srcTexture.dwPitch;
 	srcTexture.pData      = (CMP_BYTE*) lpSource;
 
-	CMP_CompressOptions options = {0};
+	CMP_CompressOptions options;
+	memset(&options, 0, sizeof(options));
+
 	options.dwSize        = sizeof(options);
-	options.fquality      = 1.0f;
 	options.dwnumThreads  = 0;
 	options.bDXT1UseAlpha = DestFormat == IMAGE_FORMAT_DXT1_ONEBITALPHA;
 
-	CMP_Texture destTexture = {0};
+	CMP_Texture destTexture;
+	memset(&destTexture, 0, sizeof(destTexture));
```

Comment:
What exactly is the `memset` accomplishing here? `CMP_Texture` looks like it has all POD types so bracket initialization should do the same thing.

Reply:
late night brain thought not, i'll revert that